lib/api/api.go

```diff
@@ -309,7 +303,7 @@ func (s *service) serve(ctx context.Context) {
 
 	// A handler that splits requests between the two above and disables
 	// caching
-	restMux := noCacheMiddleware(metricsMiddleware(getPostHandler(getRestMux, postRestMux)))
+	restMux := noCacheMiddleware(metricsMiddleware(s.addConfigHandler(getPostHandler(getRestMux, postRestMux))))
```

Comment:
Hiding the addConfigHandler under the caching rubric is sneaky. I'm unsure it even needs to be a method instead of being a section under the allready long list of handlefuncs? (I think there's some value in the list of endpoints being in one place in a long list.)